ProductListing/productListing.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <title>TheSportsAdda | Product Listing</title>
```

Comment:
You can add desc in your meta tag like author name, page content etc. Please delete the  first title

Reply:
sure.